GitUpKit/Views/GIQuickViewController.m

```diff
@@ -235,3 +251,212 @@ - (BOOL)diffFilesViewController:(GIDiffFilesViewController*)controller handleKey
 }
 
 @end
+
+#import "GICommitListViewController.h"
+@interface GIQuickViewControllerWithCommitsList () <GICommitListViewControllerDelegate>
+@property (strong, nonatomic, readonly) GICommitListViewController *leftController;
+@property (strong, nonatomic, readonly) GIQuickViewController *rightController;
```

Comment:
Let's use the names of the view controller. `commitListViewController` & `quickViewController`.